src/styles/annotator/pdfjs-overrides.scss

```diff
@@ -15,3 +15,13 @@
 .textLayer .highlight.selected {
   background-color: rgba(0, 100, 0, 0.5);
 }
+
+// Remove horizontal `margin:auto` so that PDF pages are left-aligned
+.pdfViewer .page {
+  margin: 1px 0 -8px 0;
+}
+
+// Make sure sidebar bucket bar/toolbar don't obscure PDF JS tools menu
+#toolbarViewer {
+  margin-right: 32px;
```

Comment:
Is there a variable that we could reference somewhere for this 32px value?

Reply:
I doubt it: no more than there is a variable in PDF.js' CSS source, but I'll poke around just in case to see if I can be clever.

Reply:
The magic number here is actually ours: `.annotation-frame-button` is set to `30px` (without using a SASS variable) in our own SCSS